ios/ios_jenkins_pipeline_guide.md

```diff
@@ -0,0 +1,300 @@
+# iOS Project Integration Guide
+---
+
+This guide will walk you through getting your iOS project set up on Jenkins.
+
+There is no setup you need to perform on Jenkins itself -- everything is handled via a `Jenkinsfile`, a text file named `Jenkinsfile`, that lives in the root of your repository.
+
+Jenkins is configured to scan Intrepid's Github Organization and look for projects that satisfy the requirements of being considered an iOS project. Those 2 requirements are simply:
+1. Have a repository whose name ends with `-ios`
+1. Have a `Jenkinsfile` in the root of your repo.
+
+---
+# Pipeline Basics
+
+A Pipeline is an abstract concept that describes a set of steps to go from raw code, to a delivered product. In the case of iOS projects, our pipeline handles code review, tests, and deploying binaries to our OTA server -- as well as reports, and metric collection.
+
+It's helpful to understand at a basic level what a Jenkins Pipeline is, but not strictly necessary for getting your project up and running. [Click for more information on Pipelines](https://jenkins.io/doc/book/pipeline/)
+
+Instead of having users write full Jenkinsfiles themselves, the Intrepid Pipeline Library serves as a wrapper, abstracting away all of the complexity of the build script itself, and having users just define their project model.
+
+---
+# Integration Steps
+
+#### 1. Create the Jenkinsfile
+
+The Jenkinsfile is where you define your pipeline. By default, all iOS projects on Jenkins will load the Intrepid Pipeline Library. This gives you immediate access to all the packages and classes within the library if you need to write a non-standard Jenkinsfile. Create the file by adding a text file named `Jenkinsfile` to your repo.
+
+#### 2. Define your Xcode Project model
+
+At the top of your `Jenkinsfile` (But below any library import statements), add the following line to declare your XcodeProject object:
+
+    def xcodeProject = new io.intrepid.XcodeProject()
+
+This simply creates a new XcodeProject instance, which is a [class defined in the Intrepid Pipeline library](https://github.com/IntrepidPursuits/jenkins-pipeline-library/blob/master/src/io/intrepid/XcodeProject.groovy).
+
+At the minimum, you must provide the name of your project. This should be the same name that you used when creating the Xcode project. Add this line to your Jenkinsfile
```

Comment:
I didn't want to write `workspace` because every xcode project may not have a workspace. It IS the name of your `.xcodeproj` or `.xcworkspace` -- but what's the best way to say that here. 

And shouldn't the project name never change?